source/views/settings/sections/odds-and-ends.js

```diff
@@ -28,11 +46,21 @@ class OddsAndEndsSection extends React.PureComponent<Props> {
       url: 'https://github.com/StoDevX/AAO-React-Native',
       id: 'ContributingView',
     })
+  onAppIconButton = () => this.onPressButton('IconSettingsView')
 
   render() {
     return (
       <View>
         <Section header="MISCELLANY">
+          {this.state.supported ? (
+            <Cell
```

Comment:
Please change this to `PushButtonCell` and sort the remaining props alphabetically.